Paragraph/Primary/index.jsx

```diff
@@ -54,11 +54,16 @@ Primary.propTypes = {
 }
 
 export default compose(
-  themeable((customizations, props) => ({
-    style: {
-      ...props.style,
-      color: customizations.color_text
+  themeable((customizations, props) => {
+    if (['error', 'warning'].includes(props.color)) {
```

Comment:
I really don't like this multiple return inside a conditional, but that's my personal opinion, so do with it what you will.

But rather than repeating this logic across multiple components, couldn't you do something like:

```
// themeableDefaultState.js
const UNTHEMED_STATES = ['error', 'warning']
const noTheme = function () { return {} }

const shouldTheme = function ({ color }) {
  return !UNTHEMED_STATES.includes(color)
}

export default function (adapter, props) {
  return themeable(shouldTheme(props) ? adapter : noTheme)
}

// Paragraph/Primary/index.jsx

export default compose(
  themeableDefaultState(function (customizations, props) {
    return {
      style: {
        ...props.style,
        color: customizations.color_text
      }
    }
  }, props),
  overridable(defaultStyles)
)(Primary)
```

Alternatively, if you think that the logic will actually be different between different components (maybe the prop to look at isn't `color`, or maybe the unthemed states are different, maybe just pull out the `UNTHEMED_STATES.includes(props.color)` part into a private function in the components, so that it can be made a little bit more explicit _why_ this logic is there and what it means.